docs/software/home_automation.md

```diff
@@ -24,3 +20,164 @@
+    /home/homeassistant/homeassistant-update.sh
+    ```
+
+=== "Known additional dependencies for device integration"
+
+    IKEA TRÅDFRI: `apt install autoconf`
+
+## EmonPi - Lightweight Energy usage stats with EmonPi PCB
+
+Turn your Raspberry Pi into a energy usage monitor with web interface.
+
+![DietPi home automation software EmonPi](../assets/images/dietpi-software-homeautomation-emonpi.png){: style="width:500px"}
+
+### Installation
+
+The DietPi optimized installation for EmonPi is aimed at users who want the following:
+
+- Ultra lightweight alternative installation to the official EmonPi image, with all the optimisations and features of DietPi. Allowing additional uses for your RPi device (e.g.: ownCloud server): [htop image of RPi Zero @700 MHz](https://dietpi.com/downloads/misc/EmonPi_Guide/EmonPi_DietPi_zero_700mhz.jpg), running our EmonPi installation.
+- Real time and historical energy usage statistics uploaded to the [emoncms.org](https://emoncms.org/) cloud, viewable from a web browser and the [Android App](https://openenergymonitor.org/forum-archive/node/11260.html), from anywhere in the world.
+- Use an existing RPi, or, do not wish to purchase the full EmonPi package (e.g.: case).
+- Comfortable attaching the EmonPi to your Raspberry Pi GPIO. No soldering or wiring is required.
+
+### Missing support
+
+The DietPi installation does not support:
+
+- RF transmission. RF receiver (sensor nodes) is supported (e.g.: EmonTX/EmonTH)
+- EmonPi LCD screen.
+- Local EmonCMS webserver on RPi. All EmonPi data is sent to [emoncms.org](https://emoncms.org/) cloud.
+
+### Requirements
+
+The following hardware is required:
+
+- 1x EmonPi PCB - [Order one here](https://shop.openenergymonitor.com/emonpi-shield-kit-no-enclosure/): Select 1 Clip-on CT Current Sensor (all together £33.42), or sensors based on your needs. [Image of what you will receive](https://cdn2.bigcommerce.com/server4400/98a75/product_images/optionset_rule_images/28_zoom_1429716170.jpg).  
+  This installation also supports the optional temperature sensor, but its not required.
+- 1x Raspberry Pi (any model)
+- 1x Good quality Raspberry Pi PSU. The EmonPi does not need its own power supply. It will draw power directly through the GPIOs on the RPi.
+- Alternatively you can power the RPi through the EmonPi shield via GPIO. For this, add the EmonPi PSU to the chart and skip buying a dedicated PSU for the RPi itself.
+
+=== "During Installation"
+
+    You will be asked to create a [emoncms.org](https://emoncms.org/) account and input your unique API Key. DietPi will automatically apply your API Key during installation.
+    If you did not complete this, or wish to change the API Key on your system, please follow the steps located here, otherwise continue below.
+
+=== "Attach EmonPi to RPi"
+
+    - Attach GPIO extender/riser to RPi: [Image1](https://dietpi.com/downloads/misc/EmonPi_Guide/emonPi-g2.jpg) [Image2](https://dietpi.com/downloads/misc/EmonPi_Guide/emonPi-g3.jpg)
```

Comment:
Would it be too much to actually show those images instead of linking them?

Reply:
No problem.  
Did it for the 8 referenced images.

Reply:
Also looked in the other documents whether there are similar issues: No one found. :+1: